Foundation/NSString.swift

```diff
@@ -270,7 +270,7 @@ extension NSString {
     }
     
     public func compare(string: String, options mask: NSStringCompareOptions, range compareRange: NSRange) -> NSComparisonResult {
-        return compare(string, options: [], range:compareRange, locale: nil)
+        return compare(string, options: mask, range:compareRange, locale: nil)
```

Comment:
The commit/pull request seems to be named differently than the implementation here, perhaps it should be named something along the lines of "pass mask options to locale based comparison" or something like that?